linux_os/guide/services/ssh/ssh_server/sshd_disable_user_known_hosts/ansible/shared.yml

```diff
@@ -3,13 +3,4 @@
 # strategy = restrict
 # complexity = low
 # disruption = low
-- name: "Disable SSH Support for User Known Hosts"
-  lineinfile:
-    create: yes
-    dest: /etc/ssh/sshd_config
-    regexp: ^IgnoreUserKnownHosts
-    line: IgnoreUserKnownHosts yes
-    insertbefore: ^Match
```

Comment:
This is the only thing I didn't replicate. However, I can't seem to figure out *why* this was added. This configuration value isn't affected by `Match` groups...

Reply:
Note that the bash remediation doesn't care [about this](https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/services/ssh/ssh_server/sshd_disable_user_known_hosts/bash/shared.sh).

Reply:
It appears this is the relevant bug report: https://github.com/ComplianceAsCode/content/issues/3645 

I'm guessing most of our ssh configuration parameters have this or similar problems. Should we just add this to the top of the configuration file?

Reply:
Ok, this is fixed; all SSH-based thingies are added before `Match` sections.